src/modules/consumer.js

```diff
@@ -0,0 +1,95 @@
+var parsers = require('./message-parsers'),
+  utils = require('./utils');
+
+/**
+ * Get a function to execute on incoming messages to handle RPC
+ * @param  {any} msg   An amqp.node message object
+ * @param  {string} queue The initial queue on which the handler received the message
+ * @return {function}       a function to use in a chaining on incoming messages
+ */
+function checkRpc (msg, queue) {
+  /**
+   * When message contains a replyTo property, we try to send the answer back
+   * @param  {any} _content the received message:
+   * @return {any}          object, string, number... the current received message
+   */
+  return (_content) => {
+    if (msg.properties.replyTo) {
```

Comment:
@jkernech yes they are never null, this function would not be reached if an error is thrown when generating the msg (amqplib). If I add these two checks, it will add up 2 computation on every message received, and it can go up to some thousand / sec, its not so much but since msg will be always defined it would be an useless loss